models/deforestation.go

```diff
@@ -0,0 +1,23 @@
+package models
```

Comment:
I think it is "frowned upon" in go to name packages after the tech-pattern used. It should follow a domain-driven style, where the types are located in the domain-specific package.

However, in this case, there is no other code related to the types, so..?